File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/StreamSelectionConfig.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class StreamSelectionConfig{

	@SerializedName("maxBitrate")
	private String maxBitrate;

	public void setMaxBitrate(String maxBitrate){
		this.maxBitrate = maxBitrate;
	}

	public String getMaxBitrate(){
		return maxBitrate;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/StreamSelectionConfig.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import com.google.gson.annotations.SerializedName

class StreamSelectionConfig {
    @SerializedName("maxBitrate")
    var maxBitrate: String? = null
    override fun toString(): String {
        return "StreamSelectionConfig{" +
                "maxBitrate = '" + maxBitrate + '\'' +
                "}"
    }
}